client/core/trade.go

```diff
@@ -533,7 +580,21 @@ func (t *trackedTrade) tick() (int, error) {
 			t.notify(newOrderNote("Match complete", details, db.Poke, corder))
 		}
 	}
-	return len(swaps) + len(redeems), errs.ifany()
+
+	if len(refunds) > 0 {
+		refunded, err := t.refundMatches(refunds)
+		corder, _ := t.coreOrderInternal()
+		details := fmt.Sprintf("Refunded %.8f %s on order %s",
```

Comment:
trying this out on a dcr_btc market, both refunds will say I was refunded dcr, but really one was btc, so this should use the terminology `worth` as above I think.

Reply:
Looking at the code again, this really shouldn't happen because refunds are always made to the `from` wallet, as that's the source of all client swaps. Can you recheck this?